src/index.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>CSS4 Training</title>
```

Comment:
I think it's a good idea to add here `viewport` too for responsive layout (if you do that, remember to include proper keywords in `package.json`!). Good stuff overall!

```html
<meta name="viewport" content="width=device-width, user-scalable=no">
```